back/db/migrate/20241011816395_add_autoshare_attribute_to_phase_context.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class AddAutoshareAttributeToPhaseContext < ActiveRecord::Migration[7.0]
+  def change
+    add_column :phases, :autoshare_results_enabled, :boolean, default: true, null: false
```

Comment:
@amanda-anderson This is fine to release :)

Reply:
Ok great, thanks!! :)